deploy/bundle/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
+kind: Configuration
+metadata:
+  name: config
+stages:
+- parallel: true
+  tests: []
```

Comment:
newline :smile:

Reply:
As an aside, I wonder if we should setup scorecard testing in the future. I think we can make it so that it automatically makes sure the bundle is correct, check that certain fields exists, etc

Reply:
> newline :smile: 
Blame `operator-sdk init` :D -- I'll fix.